installing/installing_aws/installing-aws-network-customizations.adoc

```diff
@@ -16,6 +16,11 @@ You must set most of the network configuration parameters during installation,
 and you can modify only `kubeProxy` configuration parameters in a running
 cluster.
 
+[IMPORTANT]
+====
+Ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For more information about the proper tags for public and private subnets, see link:https://docs.openshift.com/container-platform/4.8/installing/installing_aws/installing-aws-user-infra.html#installation-cloudformation-dns_installing-aws-user-infra[CloudFormation template for the network and load balancers] in the AWS documentation.
```

Comment:
why specify a particular version (4.8) here, I expect the same version as the current doc version. 
Also, this article is generally for IPI installation, but this note points to the UPI doc, which may confuse users.

Reply:
Done. :)